package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@hapi/joi": "16.1.7",
     "body-parser": "1.19.0",
+    "detect-indent": "^6.0.0",
```

Comment:
Minor, but we use fixed dependencies. So, `6.0.0`.

Reply:
Sorry about that, still trying to get used to. Fixed